README.md

```diff
@@ -360,6 +360,28 @@ exclude: []
 # same as --platform; SYFT_PLATFORM env var
 platform: ""
 
+# set the list of package catalogers to use when generating the SBOM
+# default = empty (cataloger set determined automatically by the source type [image or file/directory])
+# catalogers:
+#   - "ruby-gemfile"
```

Comment:
nit: do these have to be in quotes or can we just remove that part?

Reply:
I don't think so, I'll update 👍